src/frontend/efiling-frontend/src/components/page/home/__snapshots__/Home.test.js.snap

```diff
@@ -1161,6 +1161,45 @@ exports[`Home Component matches the snapshot when user cso account exists 1`] =
         </span>
       </h4>
       <br />
+      <div
+        class="bcgov-row"
+      >
+        <span>
+          Do you want to request that this submission be processed on a 
+          <b>
```

Comment:
Aren't bold tags supposedly depreciated or something? I think I asked this question before on a @yousiefc PR, and strong tags are recommended. I forget why, hopefully @yousiefc or @KDNeufeld can chime in.

Reply:
They aren't technically deprecated from what I understand. The strong tags serve to provide emphasis accessibility-wise as well, so text to speech would put emphasis on strong tags. As well not all browsers render strong the same. Some may present it as a highlight.

Reply:
so keep bold based on browser requirements?

Reply:
That's probably best for now I would say.